backend/src/main/java/org/example/backend/ReviewController.java

```diff
@@ -20,5 +19,36 @@ public class ReviewController {
     public ResponseEntity<Review> createReview (@RequestBody Map<String, String> payload) {
         return new ResponseEntity<Review>(reviewService.createReview(payload.get("reviewBody"), payload.get("imdbId")), HttpStatus.CREATED);
     }
+    // Abrufen aller Reviews
+    @GetMapping
```

Comment:
Mhh, I am not so familiar with this code but there might be a shorter way to write it ...like:

`@GetMapping()
public List<Review> getAllReviews() {
return service.getAllReviews();
}
`